empire.go

```diff
@@ -79,6 +79,9 @@ type Empire struct {
 
 	// RunRecorder is used to record the logs from interactive runs.
 	RunRecorder RunRecorder
+
+	// MessagesRequired is a boolean used to determine if messages should be required for events.
+	MessagesRequired bool
```

Comment:
I wonder if this should actually cause methods on `empire.Empire` to return something like an `empire.ErrMessageRequired`. It would end up being annoying boilerplate, because you'd have to add an `if e.MessagesRequired { return ErrMessageRequired }` to everything, but it forces consumers of the `empire` package to provide a message.

Or, if we really only want to require it from the heroku server, maybe just move this to `server/heroku` instead.